twist_mux:
  ros__parameters:
    topics:
      twist_mixer:
        topic   : mix_vel
        timeout : 0.05
        priority: 100
      keyboard:
        topic   : key_vel
        timeout : 0.1
        priority: 125
    locks:
      e_stop:
        topic   : e_stop
        timeout : 0.0
        priority: 255